public/js/helpers/date.js

```diff
@@ -1,13 +1,10 @@
-var toDateString = function (date) {
+const toDateString = date =>
   // + 1 on the month because month is zero-indexed
-  return (
-    date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate()
-  );
-};
+  `${date.getFullYear()}-${date.getMonth() + 1}- ${date.getDate()}`;
```

Comment:
Is there supposed to be an extra space before the final getDate? The old version didn't have that